site/content/in-dev/unreleased/polaris-spark-client.md

```diff
@@ -128,3 +128,21 @@ The Polaris Spark client has the following functionality limitations:
 3) Rename a Delta table is not supported.
 4) ALTER TABLE ... SET LOCATION is not supported for DELTA table.
 5) For other non-Iceberg tables like csv, it is not supported.
+
+## Iceberg Spark Client compatibility with Polaris Spark Client
+The Polaris Spark client today is not designed to be used with Iceberg Spark client together. In other words,
+there is no guarantee provided if both `org.apache.iceberg:iceberg-spark-runtime-3.5_2.12:<iceberg_version>` and 
+`org.apache.polaris:polaris-spark-3.5_2.12:<polaris_version>` are provided for the Spark `package` configuration.
+
+In order to help the usage of IcebergExtension use the same client, Polaris Spark client also ships a version of
```

Comment:
This needs some rephrasing: "In order to help the usage of IcebergExtension use the same client..."